packages/react-instantsearch/src/connectors/connectRange.test.js

```diff
@@ -470,7 +470,7 @@ describe('connectRange', () => {
           {
             label: '5 <= wot',
             attributeName: 'wot',
-            currentRefinement: { min: 5, max: undefined },
```

Comment:
so does undefined not work anymore to refine with only min?

Reply:
Nope I didn't update the `refine` behaviour. I only have updated the `currentRefinements` value provided by the `getMetadata` function.